process/charter.md

```diff
@@ -92,16 +92,16 @@ After consideration of the [charter assessment criteria](#charter-assessment-cri
 
 #### 3.2.1 Technical Strategy Team decision to initiate charter refinement {#cr-acceptance}
 
-If the Technical Strategy Team decides to initiate charter refinment, it follows these steps:
+If the Technical Strategy Team decides to initiate charter refinement, it follows these steps:
 
 Select a Chartering Facilitator
-: The Technical Strategy Team Lead selects a [Chartering Facilitator](#chartering-facilitator) and records that appointment in the [pipeline](#pipeline).
+: For the rechartering of existing groups, the Team Contact is selected by default as Chartering Facilitator and should assign the github issue to themselves. Otherwise, the [Technical Strategy Team Lead](https://www.w3.org/staff/strat/) and/or [Project &amp Process Team Lead](https://www.w3.org/staff/project/) select a [Chartering Facilitator](#chartering-facilitator) and records that appointment in the [pipeline](#pipeline) by assigning github issues.
 
 Request announcement of initiation of charter refinement
-: The [Chartering Facilitator](#chartering-facilitator) then sends a request to the W3C Communications Team to [announce the initiation of charter refinement](adv-notice.md); that document lists requirements for the announcement. In practice, if the announcement would precede the call for AC review by only a short delay, we skip the refinement phase.
+: The [Chartering Facilitator](#chartering-facilitator) then sends a request to the W3C Communications Team to [announce the initiation of charter refinement](adv-notice.md), copying the Technical Strategy Team Lead](https://www.w3.org/staff/strat/) and/or [Project &amp Process Team Lead](https://www.w3.org/staff/project/); that document lists requirements for the announcement. In practice, if the announcement would precede the call for AC review by only a short delay, we skip the refinement phase.
```

Comment:
The "and/or" here might confuse people as they may wonder which one they need to copy. I would either:
- Drop "or". Slightly more work for the charter facilitator but at least they don't have to choose.
- Only keep the Technical Strategy Team Lead. That wouldn't work so well to provide an alternate decision maker.
- Or perhaps consider creating an email alias à la `charter-leads@w3.org` but that seems a bit overkill.